modules/capnproto/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://capnproto.org/",
+    "maintainers": [],
```

Comment:
Thanks for the contribution! Are you willing to be a maintainer of this module?

Reply:
Thank you for the quick review!
Added myself now :)